src/prompts/usage-preference.ts

```diff
@@ -0,0 +1,26 @@
+import prompts from 'prompts';
+
+export type UsagePreference = 'errors' | 'style';
+
+export async function promptUsagePreference(): Promise<UsagePreference> {
+	const { preference } = await prompts({
+		type: 'select',
+		name: 'preference',
+		message: 'How would you like to use Stylelint?',
+		choices: [
+			{
+				title: 'Catch errors only',
+				description: 'Enforce only possible errors in your CSS',
+				value: 'errors',
+			},
+			{
+				title: 'Catch errors and enforce style',
+				description: 'Enforce both possible errors and stylistic conventions',
+				value: 'style',
```

Comment:
Since [we have removed the stylistic rules in version 16.0.0](https://stylelint.io/migration-guide/to-16#removed-deprecated-stylistic-rules) and standard doesn't include the [`stylelint-stylistic`](https://github.com/stylelint-stylistic/stylelint-stylistic) plugin, I find the choice of `"style"` odd.
Since it's internal it's not a big deal though.

The description could be updated to better reflect `stylelint-config-standard` intended purpose.
Sorry I don't have a suggestion :)

Reply:
I spent quite some time thinking about the wording for this. Initially, I considered using terms like 'style' or 'styleAndError,' but I wasn't entirely satisfied with how they sounded. After reviewing the descriptions of [stylelint-config-recommended](https://github.com/stylelint/stylelint-config-recommended) and [stylelint-config-standard](https://github.com/stylelint/stylelint-config-standard). Maybe I'll come up with some ideas for this later.